src/util/articleRenderers.ts

```diff
@@ -6,8 +6,20 @@ import { MWCapabilities } from '../Downloader.js'
 import { getStrippedTitleFromHtml } from './misc.js'
 import { DELETED_ARTICLE_ERROR } from './const.js'
 
+export enum ApiUrlType {
+  Unknown = 'unknown',
+  MobileRest = 'mobileRest',
+  DesktopRest = 'desktopRest',
+  VE = 've',
+}
```

Comment:
Keeping API type along with API URL might be useful for testing but not actually necessary for now since we already have a renderer builder that orchestrates with renderer processes depending on render type.